.github/workflows/deploy_app_prod.yaml

```diff
@@ -0,0 +1,76 @@
+name: Deploy App Production
+on: 
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+jobs:
+  test:
+    name: Test on Node ${{ matrix.node }} and ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: ['latest']
+        os: [ubuntu-latest]
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Use Node ${{ matrix.node }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install deps
+        run: npm ci --legacy-peer-deps
```

Comment:
Do we still need the legacy peer deps option?

Reply:
Good eye, I believe we don't